### DISCUSSION TITLE
Path is not a string & renderWithVue is not a function

### DISCUSSION THREAD

**Discussion by louie-chang** [quasarframework/quasar#12623] [CLI - SSR mode]
Hi,

I' m transferring a project into SSR, but I got these two errors and the project cannot run in local. It said:
1. `path argument is not a string`
2. `TypeError: renderWithVue is not a function`

![image](https://user-images.githubusercontent.com/91861239/156171137-54bf2c32-a508-4d4c-8912-f0104505e0b7.png)

I found a discuss about it but somehow it's 502 now. 
[https://forum.quasar-framework.org/.../try-to-build-m-ssr](https://forum.quasar-framework.org/topic/5158/try-to-build-m-ssr?fbclid=IwAR3JEH0cNBcmkpUNoB_c7eTeI8EK9snlz3_K_l01dm692HDiRzn_WAc5W84)

The Quasar version is 3.3.3 with CLI v1.22.10. I've spend a lot of time on it, so any advice would be really welcomed and apricated. If you need other information like `package.json` or `quasar.confi.js` or something, please let me know.

Thank you soo much.

**Comment by MilosPaunovic** [comment]
A code sample of some kind would be useful for people to understand your issue better and for you to get better assistance. Where do you have a function with `path` argument?

**Reply by louie-chang** [reply]
I wished I can answer that, but I cannot find where I have a function call with `path`. The error said it happened at a Vue component `AppCard.vue` inside a node module `my-module`. I didn't call function in that component, or at least I didn't know.

I created `my-module`, which is basically another Quasar project with many shared Vue components. It's already `quasar mode add ssr` 
and launched locally successfully, but may have some errors like `window is not defined`

The error showed when I enter `yarn quasar dev -m ssr`, without entering a route, and my `App.vue` doesn't contain any JavaScript code neither.

This is `AppCard.vue`. I hoped it can be a code sample. Sorry I have to mask something. If this is not enough please let me know, and I'll try to add a minimal reproducible example with Code Sandbox or something.

Thank you.

![image](https://user-images.githubusercontent.com/91861239/156194783-3987e791-a0e2-4b05-bcfb-7db00ed89135.png)
